model/UserModel.ts

```diff
@@ -1,5 +1,153 @@
+import type { UidIdentifier } from "firebase-admin/auth";
+
 export class UserModel {
-    /**
-     * Model for the user
-     */
-}
\ No newline at end of file
+    //private fields for the user's statistics
+
+    //name of the user
+    private _userName: string = '';
+
+    //current streak of the user
+    private _currentStreak: number = 0;
+    //max streak of the user
+    private _maxStreak: number = 0;
+
+    //average rank of the user
+    private _averageRank: number = 0;
+    //array of ranks of the user
+    private _ranks: number[] = [];
+
+    //average number of guesses of the user
+    private _averageGuesses: number = 0;
+    //array of number of guesses of the user
+    private _guesses: number[] = [];
+
+    //average time taken by the user
+    private _averageTime: number = 0;
+    //array of times taken by the user
+    private _times: number[] = [];
+
+    //number of times the user has played
+    private _timesPlayed: number = 0;
+
+    constructor(userName: string) {
+        this._userName = userName;
+        this._currentStreak = 12;
+        this._maxStreak = 14;
+        this._ranks = [36, 2, 12, 44, 17];
+        this._averageRank = this._ranks.reduce((a, b) => a + b, 0) / this._ranks.length;
+        this._guesses = [7, 1, 22, 9, 3];
+        this._averageGuesses = this._guesses.reduce((a, b) => a + b, 0) / this._guesses.length;
+        this._times = [14, 28, 37, 48, 50];
+        this._averageTime = this._times.reduce((a, b) => a + b, 0) / this._times.length;
+        this._timesPlayed = 5;
+    }
```

Comment:
Maybe do a mean function in Utils and use it here

Reply:
It will be removed with the next issue (#29). We will store the average inside the User DB.

Reply:
So how do you plan on updating the mean with a new value ? Maybe do a helper function updateMean then cause it's an operation you will do a few times